pallets/nfts/src/features/transfer.rs

```diff
@@ -139,6 +149,15 @@ impl<T: Config<I>, I: 'static> Pallet<T, I> {
 				return Ok(())
 			}
 
+			// All collection approvals need to be removed because otherwise
```

Comment:
Reasses based on understanding of collection approvals.

Reply:
Removed in https://github.com/r0gue-io/pop-node/pull/395